package.json

```diff
@@ -2,11 +2,12 @@
   "name": "gg-practicum-team-4-back",
   "version": "1.0.0",
   "description": "Back-End Repo for Team 4 of Giraffe/Grackle Practicum",
-  "main": "app.js",
+  "main": "src/app.js",
   "scripts": {
-    "dev": "nodemon src/server.js",
+    "dev": "node src/server.js",
     "format": "prettier --write .",
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "start": "node src/server.js"
```

Comment:
do we really need to run "node src/server.js" with `dev` and with `start `commands?